cdci_data_analysis/analysis/instrument.py

```diff
@@ -486,6 +489,14 @@ def get_product_query_name(self, product_type):
         else:
             return self.query_dictionary[product_type]
 
+
+    def check_instrument_access(self, roles, email):
+        if self.restricted_access:
+            return self.instrumet_query.check_instrument_roles(roles, email)
```

Comment:
`check_instrument_roles` also can be renamed

Reply:
any ideas on how ?

Reply:
`check_instrument_access`